spec/client_spec.rb

```diff
@@ -659,6 +659,12 @@ def loop_times(times)
     expect(resp['error']).to include 'invalid SQS url'
   end
 
+  it 'check_sns with an invalid queue url should fail' do
+    resp = @client.check_sns('key', 'secret', 'arn:aws:sns:us-east-1:123456789012:sns-topic')
```

Comment:
This looks like a pretty realistic SNS topic ARN. Is that intentional? At least the SQS one above has a clearly invalid queue URL

Reply:
The structure is a valid structure, otherwise a different error will be returned, but obviously this is a non existing one (look at the AWS account number: `123456789012`